catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -164,14 +169,23 @@ public UpdateStorageResponse version(UpdateStorageRequest streamUpdateRequest,
         setSkipFlag(streamUpdateRequest);
         setSkipFlag(updateStorageResponse);
 
-        Collection<ReadStorageRequest> ids = getReadStorageRequests(updateStorageResponse);
+        List<Metacard> updatedMetacards = updateStorageResponse.getUpdatedContentItems()
+                .stream()
+                .filter(ci -> ci.getQualifier() == null || ci.getQualifier()
```

Comment:
Would you also want (in the unlikely chance it exists) to get `"   "`, etc.? If so, can simplify this to
`.filter(ci -> StringUtils.isBlank(ci.getQualifier()))`